examples/iam/user_invite/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	bluemix "github.com/IBM-Cloud/bluemix-go"
+	v2 "github.com/IBM-Cloud/bluemix-go/api/iam/iamv2"
+	"github.com/IBM-Cloud/bluemix-go/models"
+	"github.com/IBM-Cloud/bluemix-go/session"
+	"github.com/IBM-Cloud/bluemix-go/trace"
+)
+
+func main() {
+
+	const (
+		Member = "MEMBER"
+	)
+	var userEmail string
+	flag.StringVar(&userEmail, "userEmail", "", "Email of the user to be invited")
+
+	var accountID string
```

Comment:
Is it IBMId or accountID?

Reply:
@hkantare accountID of the inviter is used.